src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -37,13 +38,34 @@ void setUserStore(UserStore userStore) {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
-        request.getRequestDispatcher("/WEB-INF/view/profile.jsp").forward(request,response);
+        //Checks whether user is logged in or not
+        String userName = (String) request.getSession().getAttribute("user");
+        System.out.println("The user store is " + userStore);
```

Comment:
Please remove all the print lines before merging to master. Usually they are used for debugging so they should be cleaned up before a merge.